### ISSUE TITLE
Update task url on Gitpay that was edited on bitbucket

### ISSUE THREAD

**Issue by alexanmtz** [worknenjoy/gitpay#515]
When the user edits some task on bitbucket repository, and this task already was added on Gitpay, the system doesn't recognize this task, because the link that was put on Gitpay is associated with the task name that was created

### Level
Beginner

### How to reproduce the user scenario
 - Create a user
 - Import an issue from bitbucket
 - Change title on Bitbucket
 - Check the issue on Gitpay

### Tips
You will need to create your own issue on Bitbucket to be able to edit and test this issue.

**Comment by nrrao** [comment]
I want to work on this issue.

**Comment by alexanmtz** [comment]
Hello @nrrao , you're now assigned and you can start to work on this issue 👍

**Comment by nrrao** [comment]
I started working on the issue, I was able to start the server and frontend. When I try to create user there was an error to create the user.
**TypeError: models.User.generateHash is not a function**
    at Object.userBuilds (C:\Users\16102\thinkfulProjects\gitpay\modules\users\userBuilds.js:5:41)
    at Object.tryCatcher (C:\Users\16102\thinkfulProjects\gitpay\node_modules\bluebird\js\release\util.js:16:23)
    at Object.userBuilds (C:\Users\16102\thinkfulProjects\gitpay\node_modules\bluebird\js\release\method.js:15:34)
    at C:\Users\16102\thinkfulProjects\gitpay\modules\app\controllers\auth.js:12:10

**Comment by nrrao** [comment]
Solved the above issue by changing sequelize version from 5  to 3.

The next problem I am facing is about connecting to Bitbucket from Gitpay.

I have created my own Bitbucket account and have configured ID and Key in the .env file.
However, I get the below error when I click "Bitbucket" button:

Invalid client_id
This integration is misconfigured. Contact the vendor for assistance

Can you send me an example of how to configure connecting to Bitbucket from Gitpay?

**Comment by alexanmtz** [comment]
@nrrao Are you following the instructions to copy the `.env.example` to `.env` in order to setup a basic environment variables set?

**Comment by nrrao** [comment]
I had set up basic environment variables set by copying .env.example to .env.
I was trying to import issue by logging into bitbucket through gitpay. I didn't have to do this!
Now I am able to add task from bitbucket.

**The issue is NOT reproducible.**
 When I changed the issue name on bitbucket it does not reflect on gitpay, but the link is still working. So the Bitbucket tasks are correctly recognized in Gitpay.
